### DISCUSSION TITLE
Searching for POI

### DISCUSSION THREAD

**Discussion by tomhanax** [osmandapp/OsmAnd#17668] [Q&A] [answered]
Using OsmAnd~ from F-droid.
I want to understand how offline search is working. Let me demonstrate on specific case.
After installation I downloaded map for Slovakia, I do not use online maps nor any online data (no data on the way).
I want to see for specific POI, node 9151211710. You can find it at https://www.openstreetmap.org/node/9151211710
So I move map to the point where it should be. But there is no bench, no icon. So I click search icon and enter "Gardeon". No results within 10km, 50km... and so on. No results at all.

I am confused. The node surely exists in the database. It is missing in offline map data? If yes, then why? If no, what is the problem?

**Comment by tommycrock** [comment]
My guess would be that osmand's map creator was confused by the dual tagging of amenity=bench and tourism=attraction. You can see that if you search for lavicka from the same spot you'll find benches and tourist attractions with that in the name.

**Reply by tomhanax** [reply]
I still don't get it. Why I can see all the benches or tourist attractions but not this specific one?

**Reply by tommycrock** [reply]
It's a guess on my part that osmand doesn't know what to do with it because it has categories for each of these and doesn't expect them to be used together. But you'd have to delve into the code or wait for someone more knowledgeable to know if that's right or why it is.

**Reply by tomhanax** [reply]
In fact I'd like to see some code, can you give some pointers where to start?

**Reply by vshcherb** [reply]
It's possibly a bug in rendering_types.xml tags are transformed from
"tags={amenity=bench, name=MEGA Lavička GARDEON, tourism=attraction, url=https://gardeon.sk/blog/lavicka-gardeon-co-stoji-za-jej-vznikom/}" to tags
"{name=MEGA Lavička GARDEON, url=https://gardeon.sk/blog/lavicka-gardeon-co-stoji-za-jej-vznikom/, bench=yes, tourism=yes}"

**Reply by tomhanax** [reply]
I see. So should I create issue, or close this or do something else?

**Reply by vshcherb** [reply, marked as answer]
I think it's an issue that should be fixed
